com.unity.shadergraph/Editor/GraphDeltaRegistry/FunctionDefinitions/StandardDefinitions/FloatNode.cs

```diff
@@ -0,0 +1,24 @@
+using System.Collections.Generic;
+using static UnityEditor.ShaderGraph.Registry.Types.GraphType;
+
+namespace com.unity.shadergraph.defs
+{
+
+    internal class FloatNode : IStandardNode
```

Comment:
Let's remove this node since we're going to make them with the new UI.

Reply:
removed float node